app/views/trln/_submast.html.erb

```diff
@@ -0,0 +1,5 @@
+<div id="trln-banner">
+  <div class="container">
+    <%= link_to t('blacklight.home.trln_submast_html').html_safe,'/trln', class: 'trln-home-link' %>
```

Comment:
`search_action_url()` should generate the correct URL to /catalog or /trln depending on your current controller context.

Reply:
Noted. Though in this particular case I always want the link in the TRLN banner to go to /trln.

Reply:
I still think it's better not to hard code paths inside the application. The helper will always return the right path to the route even if the path name changes.

Reply:
@corylown Good point -- will revise the latest commit w/this change.

Reply:
thanks!